src/components/admin/Billing/index.tsx

```diff
@@ -67,10 +92,65 @@ function AdminBilling() {
         setHydrationErrorsExist,
         billingStats,
         error,
-        hydrated,
         isValidatingStats,
     ]);
 
+    useEffect(() => {
+        if (
+            selectedTenant &&
+            dateRange.length > 0 &&
+            !hydrated &&
+            !historyInitialized
+        ) {
+            setHistoryInitialized(true);
+
+            void getBillingHistory(selectedTenant, dateRange).then(
```

Comment:
Anything driving including the `void` here? Since we have the `then` that shouldn't be needed.

Reply:
Of course not. This is a classic instance of an oversight, intensified when programming without vision.